frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -107,7 +109,7 @@ const deleteDelegatedAdmin = (
             />
             <DataTable
                 v-model:filters="delegatedAdminFilters"
-                :value="props.delegatedAdmins"
+                :value="compareDelegatedAdminTable(props.delegatedAdmins)"
```

Comment:
so instead of comparing the values, could we set the style for the rows, where row index is less or equal to the length of the new user access? like for example the top 3 rows are the new access with the specific style